store/tikv/region_request.go

```diff
@@ -95,7 +102,13 @@ func (s *RegionRequestSender) SendReqCtx(bo *Backoffer, req *tikvrpc.Request, re
 	})
 
 	for {
-		ctx, err := s.regionCache.GetRPCContext(bo, regionID)
+		var ctx *RPCContext
+		var err error
+		if sendToFlash {
+			ctx, err = s.regionCache.GetFlashRPCContext(bo, regionID)
+		} else {
+			ctx, err = s.regionCache.GetRPCContext(bo, regionID)
```

Comment:
does `GetRPCContext` also need filter out `isFlash == true`'s store at the same time?

Reply:
Now it doesn't need. But if region request can be sent to a follower. That need be filtered out.